### PR TITLE
hotfix for #128

### PR DESCRIPTION
**Pull request by henderkes**

### CODE REVIEW

**Review by Drakulix — DISMISSED** [review]
Looks good!

**Review by Drakulix — DISMISSED** [review]

**Comment by henderkes** [comment]
@Leats please test if this works

File: backend/flask_api.py [hunk 1]
```diff
@@ -73,17 +73,19 @@ def add_project():
...
+
+            else:
+                return make_response("Error: empty POST body", 400)
 
             return jsonify(return_ids)
         except ApiException as e:
             raise e
         except Exception as err:
-            return make_response("error: " + str(err), '500')
+            return make_response("Error: " + str(err), 500)
```
Comment (Drakulix) [inline backend/flask_api.py]: This should not be necessary. An uncatched exception always results in a 500 with the appropriate message, please remove.

**Review by Drakulix — CHANGES_REQUESTED** [review]

**Review by Leats — APPROVED** [review]
test_empty_post does pass, doesn't let any other previously successful tests fail. Looks good!

**Review by Drakulix — APPROVED** [review]

**Pull request merged**